### PR TITLE
[Edit] Arrays: Added codebyte for .toSorted()

### LINKED ISSUE #4137: [Edit] Arrays: Adding codebyte for `.toSorted()`
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

[PLEASE DO NOT WORK ON THIS ISSUE UNTIL YOU HAVE BEEN ASSIGNED] We would like an edit to the toSorted function entry. The term entry lives in docs/content/javascript/concepts/arrays/terms/toSorted/toSorted.md.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add a codebyte which shows how we can use `.toSorted()`.
2) You can be creative with the codebyte snippet but make sure you don't deviate a lot from the entry.

### Codebyte Example

image

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by andersooi**
### Description

- Added a codebyte for Javascript Array's toSorted() method
- Closes #4137

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@andersooi Thanks for the contribution. For your Information the .toSorted() method only works with Node.js Version 20 or above and our compiler doesn't support that. If we execute this code in the codebyte it gives an error. I discussed it with the internal team and decided to make your contributed code a regular example. Please make the suggested change.

**Comment by andersooi** [comment]
Hi @avdhoottt noted on the compatibility issue. The original entry already has an example section from the original contributor. Should I park the codebyte example (code + expected output) under that section instead?

**Comment by avdhoottt** [comment]
Edit the other example's heading as `Example 1`, `Example 2`, and then mark yours accordingly.

File: content/javascript/concepts/arrays/terms/toSorted/toSorted.md [hunk 1]
```diff
@@ -102,3 +102,37 @@ Which outputs:
 Numbers Ascending: -3,1,1,2,3,4,9,200,959,2159.2
 Numbers Descending: 2159.2,959,200,9,4,3,2,1,1,-3
 ```
+
+## Example 2
+
+The following example demonstrates the use of an arrow function in the `toSorted()` method:
```
Comment (avdhoottt) [inline content/javascript/concepts/arrays/terms/toSorted/toSorted.md:108]:
```suggestion
The following example demonstrates the use of an arrow function in the `.toSorted()` method:
```

**Review by avdhoottt — APPROVED** [review]
Looks good to me!! Thanks for the changes.

**Comment by andersooi** [comment]
@ishg-153 do let me know if there are any changes that need to be made!

File: content/javascript/concepts/arrays/terms/toSorted/toSorted.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ array.toSorted(compareFn);
 array.toSorted(function compareFn(a, b) { /* … */ });
 ```
 
-## Example
+## Example 1
```
Comment (ishg-153) [inline content/javascript/concepts/arrays/terms/toSorted/toSorted.md:49]:
```suggestion
## Example
```

File: content/javascript/concepts/arrays/terms/toSorted/toSorted.md [hunk 3]
```diff
@@ -102,3 +102,37 @@ Which outputs:
 Numbers Ascending: -3,1,1,2,3,4,9,200,959,2159.2
 Numbers Descending: 2159.2,959,200,9,4,3,2,1,1,-3
 ```
+
+## Example 2
```
Comment (ishg-153) [inline content/javascript/concepts/arrays/terms/toSorted/toSorted.md:106]:
```suggestion
```

**Review by ishg-153 — APPROVED** [review]
Hey @andersooi! Your example looks great! I have suggested minor changes. If we follow the `term-entry-template` there should just be one `## Example` sub-heading, so I have asked you to undo that.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
The above changes are requested. (Accidentally approved beforehand)

**Pull request merged**